crates/swc_ecma_transforms_compat/src/es2015/classes/constructor.rs

```diff
@@ -251,7 +258,7 @@ impl ConstructorFolder<'_> {
                 }));
                 // _getPrototypeOf(Foo.prototype)
                 let get_proto = Box::new(Expr::Call(CallExpr {
-                    span,
+                    span: span.take(),
```

Comment:
Can you change this to `span: *span`?
`Span` implements `Copy` so `.take()` is redundant.